adapter/datadog/logger_test.go

```diff
@@ -0,0 +1,70 @@
+package datadog
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	coopLogger "github.com/coopnorge/go-logger"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+)
+
+func TestSetup(t *testing.T) {
+	logger, err := NewLogger(WithGlobalLogger())
+	ddtrace.UseLogger(logger)
+	require.NoError(t, err)
```

Comment:
NIT: error-checking should be done before using the `logger` field.

Reply:
fixed